ddns-ipv64.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0"?>
+<Container version="2">
+  <Name>ddns-ipv64</Name>
+  <Repository>alcapone1933/ddns-ipv64:latest</Repository>
+  <Registry>https://hub.docker.com/r/alcapone1933/ddns-ipv64</Registry>
+  <Network>bridge</Network>
+  <Privileged>false</Privileged>
+  <Project>https://github.com/alcapone1933/docker-ddns-ipv64</Project>
+  <Overview>This Docker container is a DDNS updater for Free DynDNS - ipv64.net.&#xD;
+&#xD;
+If the ipv4 address at the site is changed, the new ipv4 address is sent to ipv64.net as an A-Record.</Overview>
+  <Category>Tools: Network:DNS</Category>
+  <Icon>https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQasFPd0QKbbYpPvuUG0EXhoVFF-Kw2XcdqDg&amp;amp;s</Icon>
+  <Requires>List of all Parameters that can be configured https://github.com/alcapone1933/docker-ddns-ipv64?tab=readme-ov-file#env-parameter</Requires>
+  <Config Name="Timezone" Target="TZ" Default="Europe/Berlin" Mode="" Description="Timezone" Type="Variable" Display="always" Required="true" Mask="false"/>
```

Comment:
A `TZ` timezone variable automatically gets passed into every Docker container on Unraid, so no need to include it in the configuration.
```suggestion
```